### PR TITLE
Various small UI changes

### PR DESCRIPTION
**Pull request by Tilka**
Feel free to pick and choose.

### CODE REVIEW

File: dwarview.glade [hunk 1]
```diff
@@ -54,19 +54,7 @@ Author: Namhyung Kim <namhyung@gmail.com>
             <property name="can_focus">False</property>
             <property name="layout_style">end</property>
             <child>
-              <object class="GtkButton" id="about_ok">
-                <property name="label">gtk-ok</property>
-                <property name="visible">True</property>
-                <property name="can_focus">True</property>
-                <property name="receives_default">True</property>
-                <property name="use_stock">True</property>
-                <signal name="clicked" handler="gtk_widget_hide" object="about" swapped="yes"/>
```
Comment (namhyung) [inline dwarview.glade:60]: I didn't notice the warning.  Can you put this on the "Close" button instead?

**Review by namhyung — COMMENTED** [review]
Thanks for your contribution!  Can you make a little more modification in the About dialog?

File: dwarview.glade (hunk 1 above)
Comment (Tilka) [reply dwarview.glade:60]: By 'this' you mean the signal handler? Closing the dialog already works, presumably because of some default.

Comment (namhyung) [reply dwarview.glade:60]: Hmm.. it didn't work for me.  I'll take a look at it later.

**Pull request merged**